cumulusci/cumulusci.yml

```diff
@@ -562,6 +562,13 @@ tasks:
         class_path: cumulusci.tasks.metadata_etl.SetOrgWideDefaults
         options:
             namespace_inject: $project_config.project__package__namespace
+    strip_unwanted_components:
```

Comment:
I think it's fine to have this as a task. In some cases, folks might want to use it explicitly against a repo.